### DISCUSSION TITLE
Enable Bitcode

### DISCUSSION THREAD

**Discussion by AlonRom** [xamarin/Xamarin.Forms#11317] [General]
I received a size limit warning when releasing Xamarin.Forms app for iOS.

Why is the option to "Enable Bitcode" is not available in Xamarin.Forms?

what is the suggested way to configure the Build when releasing a Xamarin.Forms app targeting iOS?

What are the advantages and disadvantages of using:
1. "Use LLVM optimizing compiler"
2. "Use Thumb-2 instruction set for ARMv7 and ARMv7s"

**Comment by EmilAlipiev** [comment]
LLVM is optimizing your code and supposed to make it more performant but to be honest it is minimal you wont even feel it. According to my experience it increases your app size on Android as well (not sure on IOS). you wont need it in general just use AOT even better Start up AOT on Android. Did you use linker to reduce your app size?

**Comment by AlonRom** [comment]
@EmilAlipiev it is said that LLVM suppose to reduce the app size, do you know if that's true?
why can't we Enable Bitcode?
